src/app/models/work-item-type.ts

```diff
@@ -89,8 +83,8 @@ export class WorkItemTypeMapper implements Mapper<WorkItemTypeService, WorkItemT
         toPath: ['type'],
         toValue: 'workitemtypes'
       }, {
-        fromPath: ['relationships', 'infotip', 'data'],
-        toPath: ['infotip'],
+        fromPath: ['attributes', 'description'],
+        toPath: ['description'],
```

Comment:
As we are sure of description coming from from the backend, do we need to give a default text anymore? We can still check for `null` and return empty string on `null`.

Reply:
Yes we can do that.

Reply:
Why not a placeholder text like "coming soon..." instead of empty info-tip that may look like an error. Let's see what UX has to suggest. @Veethika